src/main.rs

```diff
@@ -85,18 +85,22 @@ fn main() {
 
     let inside_config = miniserve_config.clone();
 
-    let interfaces = miniserve_config.interfaces.iter().map(|&interface| {
-        if interface == IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)) {
-            // If the interface is 0.0.0.0, we'll change it to localhost so that clicking the link will
-            // also work on Windows. Why can't Windows interpret 0.0.0.0?
-            String::from("localhost")
-        } else if interface.is_ipv6() {
-            // If the interface is IPv6 then we'll print it with brackets so that it is clickable.
-            format!("[{}]", interface)
-        } else {
-            format!("{}", interface)
-        }
-    });
+    let interfaces = miniserve_config
+        .interfaces
+        .iter()
+        .map(|&interface| {
+            if interface == IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)) {
+                // If the interface is 0.0.0.0, we'll change it to 127.0.0.1 so that clicking the link will
+                // also work on Windows. Why can't Windows interpret 0.0.0.0?
+                String::from("127.0.0.1")
```

Comment:
I thought nowadays you're supposed to do `.to_string()` so let's do that here.

Reply:
I'll fix this asap